zipkin/src/test/java/zipkin/TestObjects.java

```diff
@@ -65,6 +65,8 @@
           .addAnnotation(Annotation.create((TODAY + 100) * 1000, SERVER_RECV, APP_ENDPOINT))
           .addAnnotation(Annotation.create((TODAY + 250) * 1000, SERVER_SEND, APP_ENDPOINT))
           .addAnnotation(Annotation.create((TODAY + 300) * 1000, CLIENT_RECV, WEB_ENDPOINT))
+          .addAnnotation(Annotation.create((TODAY + 350) * 1000, SERVER_SEND, DB_ENDPOINT))
```

Comment:
If you are looking to create multiple annotations, I might do so in a different file as this one is used for all storage tests (and also some non-storage ones). Particularly, this creates an invalid "party-line" span, which has multiple servers for the same RPC.